lib/trino-parquet/src/test/java/io/trino/parquet/TestParquetTimestampUtils.java

```diff
@@ -42,7 +42,7 @@ public void testInvalidBinaryLength()
     {
```

Comment:
Please update the commit message "Fix more deprecated Parquet API usage" to not use "more" as that implies certain ordering to the commits which may or may not exist.

Reply:
Done